packages/web/src/scss/components/Dropdown/_Dropdown.scss

```diff
@@ -1,4 +1,5 @@
 @use 'sass:map';
+@use '../../settings/featureFlags';
```

Comment:
I'm afraid we use kebab-case for naming non-component stylesheets, see `_scroll-control.scss` or `_form-fields.scss`.